.travis.yml

```diff
@@ -60,6 +60,13 @@ matrix:
         - TESTMODE=full
         - COVERAGE=--coverage
         - NUMPYSPEC=numpy
+  allow_failures:
+    - python: 2.7
+      env:
+        - TESTMODE=fast
+        - COVERAGE=
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@master"
+
```

Comment:
I believe python 3.5 CI run uses numpy master (or a wheel built from it, which should be just the same.)